### ISSUE TITLE
PR-CI service killed for unkwon reasons

### ISSUE THREAD

**Issue by netoarmando** [freeipa/freeipa-pr-ci#309]
PR-CI hits a `State 'stop-sigterm' timed out` error and it's stopped after that.

The first two lines of the following log are not related, they just show that PR-CI was in IDLE mode, after checking errors in some PR definitions.
```
Sep 17 17:10:14 small-runner-23 bash[2567]: ERROR: __main__: string indices must be integers
Sep 17 17:10:14 small-runner-23 bash[2567]: ERROR: __main__: string indices must be integers
Sep 17 17:11:32 small-runner-23 systemd[1]: prci.service: State 'stop-sigterm' timed out. Killing.
Sep 17 17:11:32 small-runner-23 systemd[1]: prci.service: Killing process 2567 (prci.py) with signal SIGKILL.
Sep 17 17:11:32 small-runner-23 systemd[1]: prci.service: Main process exited, code=killed, status=9/KILL
Sep 17 17:11:32 small-runner-23 systemd[1]: prci.service: Failed with result 'timeout'.
Sep 17 17:11:32 small-runner-23 systemd[1]: Stopped FreeIPA PR CI.
```
Manual intervention is required after that.

**Comment by netoarmando** [comment]
It can be related to this: https://github.com/freeipa/freeipa-pr-ci/blob/61128003de778ef12672166ff58fada44e47640b/tasks/common.py#L100-L111

**Comment by netoarmando** [comment]
Another error with similar behaviour: https://gist.github.com/netoarmando/d3a7bc1eda297b9df78399ce72ca53ac
```bash
Oct 01 19:12:04 big-runner-4 bash[1034]: DEBUG: root: RunADTests timed out after 4800s
Oct 01 19:12:04 big-runner-4 bash[1034]: Traceback (most recent call last):
Oct 01 19:12:04 big-runner-4 bash[1034]:   File "/root/freeipa-pr-ci/tasks/common.py", line 124, in __call__
Oct 01 19:12:04 big-runner-4 bash[1034]:     super(FallibleTask, self).__call__()
Oct 01 19:12:04 big-runner-4 bash[1034]:   File "/root/freeipa-pr-ci/tasks/common.py", line 108, in __call__
Oct 01 19:12:04 big-runner-4 bash[1034]:     raise TimeoutException(self)
Oct 01 19:12:04 big-runner-4 bash[1034]: tasks.common.TimeoutException: RunADTests timed out after 4800s
Oct 01 19:12:04 big-runner-4 bash[1034]: DEBUG: raven.base.Client: Configuring Raven for host: https://sentry.io
Oct 01 19:12:04 big-runner-4 bash[1034]: DEBUG: raven.base.Client: Sending message of length 1407 to https://sentry.io/api/193222/store/
Oct 01 19:12:18 big-runner-4 bash[1034]: INFO: __main__: Available resources: 8 CPU, 15532.203125MB
Oct 01 19:13:34 big-runner-4 systemd[1]: prci.service: State 'stop-sigterm' timed out. Killing.
Oct 01 19:13:34 big-runner-4 systemd[1]: prci.service: Killing process 1034 (prci.py) with signal SIGKILL.
Oct 01 19:13:34 big-runner-4 systemd[1]: prci.service: Main process exited, code=killed, status=9/KILL
Oct 01 19:13:34 big-runner-4 systemd[1]: prci.service: Failed with result 'timeout'.
Oct 01 19:13:34 big-runner-4 systemd[1]: Stopped FreeIPA PR CI.
```

**Comment by netoarmando** [comment]
Closing this for now, PR #319 was merged and deployed.
